simtools/applications/db_development_tools/add_model_parameters_from_repository_to_db.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/python3
 """
-    This application adds all parameters found in a repository to the DB.
+    This application adds all parameters found in a repository to a new database.
 
-    Generates a new data with all required collections.
+    Generates a new database with all required collections.
```

Comment:
I have looked into this and I think we don't need to update it: if the database exists, it will add/update the existing database. So this is fine, as we sometimes add new parameters or updates to parameters.